rest-api/dao/participant_dao.py

```diff
@@ -72,6 +73,13 @@ def insert(self, obj):
     assert not obj.biobankId
     return self._insert_with_random_id(obj, ('participantId', 'biobankId'))
 
+  def _check_if_external_id_exists(self, obj):
+    with self.session() as session:
+      participant = session.query(Participant).filter_by(externalId=obj.externalId).first()
```

Comment:
You can just return this. You don't need the if below.

Reply:
done